package.json

```diff
@@ -24,7 +24,7 @@
   },
   "homepage": "https://github.com/timocov/ts-transformer-minify-privates#readme",
   "peerDependencies": {
-    "typescript": ">=2.9.1"
+    "typescript": ">=4.8.2"
```

Comment:
is it the first version of when the api you're using is supported? it looks like it is big breaking change in terms of how much major and minor versions we're bumping here

Reply:
maybe we can play around and create wrappers that will handle this depending on whether an API is supported or not?

Reply:
> maybe we can play around and create wrappers that will handle this depending on whether an API is supported or not?

done

Reply:
Many thanks!